services/Bundler.js

```diff
@@ -26,7 +27,8 @@ module.exports = class Bundler {
         .then(installPackages({ id, cwd: targetDir }))
         .then(createBundle({ id, cwd: targetDir }))
         .then(cacheBundle(id))
-        .then(notifyListeners(id));
+        .then(notifyListeners(id))
+        .then(result => result, reportError);
```

Comment:
adding simple error reporter for errors in promises here. hopefully this will help us narrow down the issue